ansible/roles/fluentd-node-collector/tasks/deploy_logging.yaml

```diff
@@ -20,10 +20,10 @@
 - name: "Process roles template and create objects"
   shell: "oc process fluentd-roles-template | oc create -f - "
 
-- name: "Import fluentd image as imagestream"
-  shell: >
-    oc import-image logging-fluentd --from=docker.io/t0ffel/logging-fluentd
-    --all --confirm
+# - name: "Import fluentd image as imagestream"
```

Comment:
imagestreams are imho preferential in general to use, although it didn't work for me.. I think there are some weird bugs there.